mappings/net/minecraft/world/World.mapping

```diff
@@ -49,10 +49,20 @@ CLASS net/minecraft/class_1937 net/minecraft/world/World
 		ARG 1 mapState
 	METHOD method_17891 getMapState (Ljava/lang/String;)Lnet/minecraft/class_22;
 		ARG 1 id
-	METHOD method_18023 getEntities (Lnet/minecraft/class_1299;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+	METHOD method_18023 getEntitiesByType (Lnet/minecraft/class_1299;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+		COMMENT Computes a list of entities of the given type within some region.
+		COMMENT
+		COMMENT <strong>Warning:<strong> If {@code null} is passed as the entity type filter, care should be
+		COMMENT taken that the type argument {@code T} is set to {@link Entity}, otherwise heap pollution
+		COMMENT or bad casts can occur.
+		COMMENT
+		COMMENT @return a list of entities of the given type
```

Comment:
of the given type (if a non-null type is given) and matching the predicate!

Reply:
just `a list of entities`